src/app/screens/__tests__/AddAccount.test.tsx

```diff
@@ -293,106 +304,18 @@ describe('AddAccount', () => {
   });
 
   it('shows mnemonic crypto error', async () => {
-    const { getByLabelText, getByText, getByTestId } = getComponent(createStore());
-    const mnemonicButton = getByTestId('select-MNEMONIC');
-    expect(mnemonicButton).toBeDefined();
-    fireEvent.click(mnemonicButton);
-
-    await waitFor(() => expect(getByTestId('mnemonic-input')).toBeDefined());
-
-    const mnemonicInput = getByTestId('mnemonic-input');
-    expect(mnemonicInput).toBeDefined();
-    fireEvent.change(mnemonicInput, { target: { value: fMnemonicPhrase } });
-
-    const passwordInput = getByLabelText(translateRaw('MNEMONIC_PASSWORD'));
-    fireEvent.change(passwordInput, { target: { value: 'password' } });
-
-    ipcBridgeRenderer.crypto.invoke = jest.fn().mockImplementation(() => {
-      throw new Error('error');
+    const mockStore = createMockStore({
+      accounts: {
+        addresses: [],
+        fetchError: 'error'
+      }
     });
 
-    const submitButton = getByText(translateRaw('NEXT'));
-    expect(submitButton).toBeDefined();
-    fireEvent.click(submitButton);
-
-    await waitFor(() => expect(getByText('error', { exact: false })).toBeDefined());
-  });
-
-  it('can submit mnemonic from another DPath and page', async () => {
```

Comment:
I would prefer if we could keep this test. Would it be possible to run it with a regular store to fix the problems you've had?